src/new_index/schema.rs

```diff
@@ -1375,18 +1432,6 @@ impl TxHistoryRow {
 impl TxHistoryInfo {
     // for funding rows, returns the funded output.
     // for spending rows, returns the spent previous output.
-    pub fn get_funded_outpoint(&self) -> OutPoint {
-        match self {
-            TxHistoryInfo::Funding(ref info) => OutPoint {
-                txid: deserialize(&info.txid).unwrap(),
-                vout: info.vout as u32,
-            },
-            TxHistoryInfo::Spending(ref info) => OutPoint {
-                txid: deserialize(&info.prev_txid).unwrap(),
-                vout: info.prev_vout as u32,
-            },
-        }
-    }
 }
```

Comment:
With this change, the implementation of the `impl TxHistoryInfo {}`block is empty and can be removed?